src-java/base-topology/base-messaging/src/test/java/org/openkilda/messaging/info/switches/v2/LogicalPortsValidationEntryV2Test.java

```diff
@@ -42,6 +42,22 @@ public void splitAndUniteEmptyLogicalPortEntryTest() {
         assertEquals(entry, united);
     }
 
+    @Test
+    public void splitAndUniteEmptyLogicalPortEntryWithErrorTest() {
+        LogicalPortsValidationEntryV2 entry = LogicalPortsValidationEntryV2.builder()
+                .asExpected(false)
+                .error("Timeout for waiting response on DumpLogicalPortsRequest() Details: Error in SpeakerWorkerService")
+                .missing(new ArrayList<>())
+                .misconfigured(new ArrayList<>())
+                .excess(new ArrayList<>())
+                .proper(new ArrayList<>())
+                .build();
+        List<LogicalPortsValidationEntryV2> list = entry.split(4, 4);
+        assertEquals(1, list.size());
+        LogicalPortsValidationEntryV2 united = LogicalPortsValidationEntryV2.join(list);
+        assertEquals(entry, united);
+    }
+
```

Comment:
I think also you should test for a case when we have list size > 1.

Reply:
thanks, added appropriate test.